CIP-0035/README.md

```diff
@@ -0,0 +1,38 @@
+---
+CIP: 35
+Title: Daedalus URI Scheme
+Authors: Daniel Main <daniel.main@iohk.io>
+Comments-URI:
+- https://github.com/cardano-foundation/CIPs/pull/130
+Status: Draft
+Type: Informational
+Created: 2022-03-18
+License: CC-BY-4.0
+---
+
+# Abstract
+
+This proposal is an extension of CIP-0013, which describes a basic URI scheme to handle ada transfers or partial transactions with Daedalus or any desktop Wallet installed in the operating system.
+
+# 💡 Motivation
+
+DEX, Web-dApps and Desktop-dApps (e.g. games or desktop applications) should have the possibility to interact directly with desktop wallets. By defining a URI scheme and registering it in the operating system, it will allow opening Daedalus or any desktop wallet same as when you call or click a mailto:email@sample.com and your email client is opened.
+
+An application could ask for a payment and offer options like PayPal, Visa, cardano-web. Although a user might want to help cardano to get more decentralized using a full-node-wallet that is why the application could also offer payments with a cardano-desktop.
+The usage of a different URL will solve the problem of users, which have multiple wallets: browser-wallet for daily usage and a secure desktop wallet for the savings.
+
+# 📖 Specification
+
+The core implementation should follow the [BIP-21](https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki) standard (with `bitcoin:` replaced with `web+cardano:`)
+
+Correct examples:
+```
+<a href="web+cardano://Ae2tdPwUPEZ76BjmWDTS7poTekAvNqBjgfthF92pSLSDVpRVnLP7meaFhVd?unix-timestamp-timeout=1647896861">Buy/Send/Donate</a>
+<a href="web+cardano-testnet://Ae2tdPwUPEZ76BjmWDTS7poTekAvNqBjgfthF92pSLSDVpRVnLP7meaFhVd?unix-timestamp-timeout=1647896861">Buy/Send/Donate (Testnet)</a>
```

Comment:
> I would strongly advise against doing `web+cardano-testnet`. Cardano, just like Ethereum, it sure to have many different networks in the future and we don't want to be stuck with a bunch of different URI schemes for every network. Instead, I suggest you include something similar to [CIP34](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0034?rgh-link-date=2022-03-24T15%3A06%3A06Z) inside the payload

@SebastienGllmt : the network cannot be "selected" in Daedalus. For each Network exist a different Daedalus installation. This is how it is implemented, and I am afraid this will not be changed.

Coming back to the vscode example. 

`vscode://` => opens Visual Studio
 `vscode-insiders://` => opens Visual Studio Insiders

<img width="306" alt="grafik" src="https://user-images.githubusercontent.com/4303099/159992617-2c4ea7d1-1c8e-4800-a9b6-f97f44b0cfbf.png">

So, Flight is for us, like the Insider version.
cc: @nikolaglumac

Reply:
> Cardano, just like Ethereum, it sure to have many different networks in the future and we don't want to be stuck with a bunch of different URI schemes for every network. Instead, I suggest you include something similar to [CIP34](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0034) inside the payload

At least in Daedalus’ case, each network is handled by a different build.

If we are to define only a single `protocol:`, by which the OS differentiates which app to choose, we would have to either:
* let the user choose, after clicking *a* link, in a system prompt, which Daedalus build to use to handle it, and
    * how easy it would be to mis-click is not for me to decide,
    * I think some OS-es don’t show the full list of possible apps, just a few recent ones?
* or create a completely separate small app that opens an appropriate Daedalus build (if it is installed) based on `protocol:THIS_PART` which the OS is not concerned with – in effect re-implementing a great deal of what already is in all 3 OS-es, and having to make it cross-platform, having to detect *where* each Daedalus is installed etc. etc. – a mess.